Add support for git grep regex error (#966)
VirtusLab/git-machete-intellij-plugin: CONTRIBUTING.md

3. Prepend the found path without `/grep` suffix to `PATH` (`/opt/homebrew/Cellar/grep/3.7/libexec/gnubin` in that case).
You may want to add the following `PATH="/opt/homebrew/Cellar/grep/3.7/libexec/gnubin:$PATH"` to (`.zprofile`/`.zshrc`)

> It is possible that after configuring the git hooks they return the following error: <br/>

Reviewer: ```suggestion
It is possible that git `pre-commit` hook will raise the following error: <br/>
```

> `fatal: cannot use Perl-compatible regexes when not compiled with USE_LIBPCRE` <br/>

Reviewer 2: this error does not appear on my machine and do not know how to get it, but I did instructions with compiling `git` with USE_LIBRCPE and they passed without any error

> `fatal: cannot use Perl-compatible regexes when not compiled with USE_LIBPCRE` <br/>

Reviewer 3: I do not remember that one - now everything works fine to me
